### ISSUE TITLE
yuck gen -Hfile.h usage.yuck

### ISSUE THREAD

**Issue by ammon0** [hroptatyr/yuck#10]
`yuck gen -Hfile.h usage.yuck`
Produces a header file that includes itself, which then fails to compile.

```
#if defined HAVE_VERSION_H
# include "version.h"
#endif	/* HAVE_VERSION_H */
#include <stdlib.h>
#include <stdbool.h>
#include <string.h>
#include <stdio.h>
#include <limits.h>
#include "file.h"  <<<<<<<<<<<<<<<<<<<<<<<<<<

#if defined __INTEL_COMPILER
# pragma warning (push)
# pragma warning (disable:177)
# pragma warning (disable:111)
# pragma warning (disable:3280)
#elif defined __GNUC__
```

**Comment by hroptatyr** [comment]
Thanks for the report!  This should be fixed in 72a883b.

**Comment by ammon0** [comment]
Looks good to me. Thanks.
